k8s/helm/templates/setup-snapshot-controller.yaml

```diff
@@ -35,7 +35,7 @@ spec:
       serviceAccountName: snapshot-controller
       containers:
         - name: snapshot-controller
-          image: gcr.io/k8s-staging-sig-storage/snapshot-controller:v5.0.0
+          image: {{ or .Values.snapshot_controller_image "gcr.io/k8s-staging-sig-storage/snapshot-controller:v5.0.0"}}
```

Comment:
I would just use the image from the values. A potential error is hard to recognize if someone accidentally removes the value because the pod will always start with the old v5.0.0 image.